AsyncDisplayKit/TextKit/ASTextKitRenderer+TextChecking.mm

```diff
@@ -75,6 +74,7 @@ - (NSTextCheckingResult *)textCheckingResultAtPoint:(CGPoint)point
     truncationTokenRange = { 0, truncationAttributedString.length };
   }
 
+  NSRange visibleRange = self.truncater.firstVisibleRange;
   truncationTokenRange.location += NSMaxRange(visibleRange);
```

Comment:
It returns `NSNotFound` not 0 ... I agree we use it mostly internally and don't have a proper check in place. I will put up a separate PR that uses 0,0 for `visibleRange`